network/src/network.rs

```diff
@@ -926,6 +1037,20 @@ impl NetworkService {
             protocol_metas.push(disconnect_message_meta);
         }
 
+        // Penetration protocol
+        if config
+            .support_protocols
+            .contains(&SupportProtocol::Penetration)
+        {
+            let feeler_meta = SupportProtocols::Penetration.build_meta_with_service_handle({
```

Comment:
1. name typo, it should `penetration_meta`
2. `build_meta_with_service_handle` is not set compress to protocol. does this protocol need to be compressed?